lib/graphql/schema/validation.rb

```diff
@@ -201,6 +214,17 @@ def self.assert_named_items_are_valid(item_name, get_items_proc)
           Rules::SCHEMA_CAN_GENERATE_IDS,
         ],
       }
+
+      INTROSPECTION_TYPES = Hash[[
```

Comment:
Lol, at first I thought, how about 

```
def introspection?
  name.start_with?("__")
end
```

😝 derp 

This seems fine for now, if it becomes a maintenance problem we can fix it then!